src/gui/gatewayer_mock_test.go

```diff
@@ -968,7 +967,7 @@ func (m *GatewayerMock) UpdateWalletLabel(p0 string, p1 string) error {
 func (m *GatewayerMock) GetDefaultStatus() *daemon.ConnectionsHealth {
 
 	connsDefault := m.GetDefaultConnections()
-	sort.Strings(connsDefault)
+	// sort.Strings(connsDefault)
```

Comment:
why is this commented

Reply:
@stdevHan is this he line you were saying that was causing test failures depending on go lang version ? What was the version that was failing ?